mgplvm/crossval/train_model.py

```diff
@@ -21,11 +21,15 @@ def training_params(**kwargs):
         'neuron_idxs': None,
         'mask_Ts': None,
         'n_mc': 32,
-        'prior_m': None
+        'prior_m': None,
+        'accumulate_gradient': True
     }
 
     for key, value in kwargs.items():
-        params[key] = value
+        if key in params.keys():
+            params[key] = value
+        else:
+            print('adding', key)
```

Comment:
What does this do? Why print 'adding key' here? Do the keys that get printed get added to params? It doesn't look like it does to me ... am I missing something?